packages/client/src/addresses/__fixtures__/getAddressPredictionDetails.fixtures.ts

```diff
@@ -1,7 +1,7 @@
 import { rest, type RestHandler } from 'msw';
 import type { AddressPrediction } from '../types/index.js';
 
-const path = '/api/account/v1/addressesprediction/:predictionId/address';
+const path = '/api-next/account/v1/addressesprediction/:predictionId/address';
```

Comment:
To ease the update of these paths in the future, I would suggest defining a constant on configs or for testing configs of the baseURL to use (`/api-next` in this case), and use it like this:

```
import { baseURL } from 'tests/<path-to-localtion>';

const path = '${baseURL}/account/v1/addressesprediction/:predictionId/address';
```

What do you think?